Formula/ipget.rb

```diff
@@ -0,0 +1,22 @@
+class Ipget < Formula
+  desc "Retrieve files over IPFS and save them locally"
+  homepage "https://github.com/ipfs/ipget/"
+  url "https://github.com/ipfs/ipget/archive/refs/tags/v0.9.1.tar.gz"
+  sha256 "d065de300a1764077c31900e24e4843d5706eb397d787db0b3312d64c94f15a9"
+  license "MIT"
+  head "https://github.com/ipfs/ipget.git", branch: "master"
+
+  depends_on "go@1.18" => :build # quic-go cannot be built with go 1.19
```

Comment:
Do we have upstream issue to track this?

Reply:
I am afraid not.

Reply:
`quic-go` started to support `go@1.19` from version 0.28 onwards: https://github.com/quic-go/quic-go/issues/3491 (and `go@1.20` since 0.32: https://github.com/quic-go/quic-go/commit/576d85cd3d80c67f30e54b6080ef7fb98732d209).

I can see that upstream has already upgraded `quic-go` on their `master` branch: https://github.com/ipfs/ipget/blob/master/go.mod#L144; and a head build with `go@1.19` succeeded. So, we should probably add a note here:

```suggestion
  # The current version of `quic-go` dependency can only be built with `go@1.18`.
  # Try `go@1.19` or newer at next release; upstream has already upgraded `quic-go` at master branch.
  depends_on "go@1.18" => :build
```